tools/cloudera/build_hue_common.sh

```diff
@@ -56,9 +56,12 @@ function redhat7_ppc_install() {
       xmlsec1-openssl \
       unzip'
     # NODEJS 14 install
```

Comment:
Update comment to explain why we use v 16

Reply:
Added a comment to explain why we are moving to v16.